### PR TITLE
Add draft version of Romps (2014) humidity model.

### PR DESCRIPTION
**Pull request by lkluft**
This PR adds a draft version of a relative humidity model described in [Romps 2014](https://journals.ametsoc.org/doi/10.1175/JCLI-D-14-00255.1).

As is, the model is unstable for the default choices of solar constant (runaway greenhouse). A reduction of the solar constant by about 10 Wm^-2 allows stable simulations.

### CODE REVIEW

File: konrad/humidity.py [hunk 1]
```diff
@@ -269,3 +270,53 @@ def __call__(self, atmosphere, convection, **kwargs):
...
+        z = atmosphere.get('z', keepdims=False)
+
+        if self._vmr_profile is None:
+            vmr = relative_humidity2vmr(
+                relative_humidity=self.get_relative_humidity_profile(T),
+                pressure=p,
+                temperature=T,
+            )
+        else:
+            vmr = self._vmr_profile
```
Comment (SallyDa) [inline konrad/humidity.py]: It's not really a problem, but why is this here? If we choose Romps14, we want to use RH(T) and not a vmr_profile specified in the initialisation.

**Comment by lkluft** [comment]
> Then we need to sort out a "better" relative humidity as a function of temperature relationship. I think using ERA5 would be nice, but as we discussed we would need to extrapolate it somehow. If you send me the data, I can have a go at this, or you can keep working on it if you'd prefer! We don't want a runaway as our reference climate!

The problem with the ERA5 data is, that the change in ``RH(T)`` is quite large at the surface. This is translated into a massive feedback in our warming model.

I tried to play around with the surface albedo a bit. One could argue that our current value of 0.2 is rather low given the fact that we do not have clouds.

**Comment by SallyDa** [comment]
How about we take just the upper part from ERA5, so we don't have such a strong spike in RH in the upper troposphere. We could get rid of the lower tropospheric peak and have RH following the idealised profile above ~300 K. In Romps Fig 7, we see that RH is not conserved with T close to the surface in the "realistic" model, so probably it also isn't in the real world, and maybe it's okay to take a simplified approach here.

I'm not so sure about your albedo arguments, as we're supposed to be modeling clear sky, which in the tropics is mainly over ocean (albedo ~0.07), so I already think our albedo is too high! Especially later if we want to look at the effect of clouds, as I guess low clouds basically only affect things through their albedo.

**Comment by lkluft** [comment]
> How about we take just the upper part from ERA5, so we don't have such a strong spike in RH in the upper troposphere. We could get rid of the lower tropospheric peak and have RH following the idealised profile above ~300 K. In Romps Fig 7, we see that RH is not conserved with T close to the surface in the "realistic" model, so probably it also isn't in the real world, and maybe it's okay to take a simplified approach here.

I am running some simulations to test a fixed relative humidity above 295 K. The ``RH(T)`` mapping for increasing temperatures above the tropopause is still very messy, but let's see how that works.

> I'm not so sure about your albedo arguments, as we're supposed to be modeling clear sky, which in the tropics is mainly over ocean (albedo ~0.07), so I already think our albedo is too high! Especially later if we want to look at the effect of clouds, as I guess low clouds basically only affect things through their albedo.

I guess it is fine to have a higher surface albedo than the 0.07 for the ocean. Even though we are looking at clear-sky, a cloud-free tropical aqua-planet is a bit too far off. So using a more "global albedo" to account for missing clouds should be fine. You are right, though, that we have to adjust that when including clouds!

**Comment by SallyDa** [comment]
> I am running some simulations to test a fixed relative humidity above 295 K. The `RH(T)` mapping for increasing temperatures above the tropopause is still very messy, but let's see how that works.

Ok, hopefully that works well. I don't think the region above the cold point should be a problem if we apply the stratospheric humidity adjustment afterwards. So I guess you don't need to care too much about `RH(T)` there.
